lein-jank/test

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Will you please move this to `lein-jank/bin/ci/test`? All of our scripts are in `bin` dirs. Anything CI specific is in `bin/ci`.

Reply:
Whoops, misread that.